Jenkinsfile

```diff
@@ -3,10 +3,10 @@ node('docker') {
         checkout scm
      }
      stage('docker-pull') {
-        sh 'docker pull fbelzunc/ad-build-container'
+        sh 'docker pull fbelzunc/ad-build-container-with-docker-fixtures'
```

Comment:
Typically you want to put the fixture inside this plugin. See other plugins which define their own fixtures for examples.

Reply:
@jglick The diagram below shows what I am trying to achieve.

As you can see:

* I am picking up a `docker` agent from jenkins-infra - which provides you access to the Docker Engine
* On this agent, I am pulling the docker image from [here](https://github.com/fbelzunc/active-directory-plugin-build-container-with-docker-fixtures). This docker image is a DIND image, which contains Maven in order to run `mvn clean verify` in the active-directory plugin. The reason why I am not using a `DockerFixture` is that I need a container where I can set-up the DNS `--dns=127.0.0.1 --dns=8.8.8.8 `.
* Maven docker-fixtures tests will run inside `active-directory-plugin-build-container-with-docker-fixtures`.

So this is the reason why I am not using `DockerFixtures`.